frontend/src/components/task/task.js

```diff
@@ -467,6 +477,14 @@ class Task extends Component {
     this.setState({ assignDialog: false })
   }
 
+  handleDeleteTask = () => {
+    this.props.deleteTask({
+      id: this.props.match.params.id,
+      userId: this.props.user.id
+    }, this.props.history)
```

Comment:
So here in the component we remove the history as parameter, and we can handle the promise this way:
```
  handleDeleteTask = () => {
    this.props.deleteTask({
      id: this.props.match.params.id,
      userId: this.props.user.id
    }).then(response => {
        history.push('/tasks/all')
       this.setState({ deleteDialog: false })
}).catch(e => console.log(e))
```
We can trigger the notifications actions here as well, but we already are doing this on the actions, se we can leave there and don't need to trigger error message on catch.

Reply:
This line `.catch(e => console.log(e))` causes prepush error: ` error  Unexpected console statement  no-console`